resources/sample_vaults/Tasks-Demo/Manual Testing/Styling of Queries.md

```diff
@@ -36,6 +36,9 @@ short mode
 ```tasks
 path includes Styling of Queries
 hide priority
+hide backlinks
+hide urgency
+hide edit button
```

Comment:
This smoke test is a little confusing now, in that the instruction says:

```
- [ ] **5. Test 'hidden' query classes** - the following lines (except the backlinks) should be colored **red**:
```

But there are no backlinks now..

Reply:
Oh damn, I fixed this, but after I did the test I checked this box and then reverted the line together with the fixed phrasing :facepalm:

Reply:
Fixed

Reply:
Been there, done that, got the t-shirt … 🤣